protos/otc/v1/types.proto

```diff
@@ -53,26 +55,26 @@ enum OrderState {
   ORDER_STATE_ERRORED = 10;
 }
 
+enum TradableSymbol {
+  TRADABLE_SYMBOL_UNSPECIFIED = 0;
+  TRADABLE_SYMBOL_SWAP = 1;
+  TRADABLE_SYMBOL_BUTTERFLY = 2;
+  TRADABLE_SYMBOL_SPREAD = 3;
+}
+
 
 enum OrderActionType {
```

Comment:
@lsbardel currently our model contains action types for updates too should they also be  included. 

```
pub enum OrderActionType {
    Unspecified = 0,
    PendingCreated = 11,
    PendingUpdate = 12,
    PendingCancel = 13,
    Created = 21,
    UpdateCreated = 22,
    CancelCreated = 23,
    Rejected = 31,
    UpdateRejected = 32,
    CancelRejected = 33,
    Fill = 41,
    FillCorrect = 42,
    FillCancel = 43,
    Expired = 51,
    Suspend = 52,
    Disconnect = 53,
    Update = 54,
}
```

Should we include all ?

Reply:
As discussed offline getting rid of OrderActionType as we no longer need it.